recipes/r-vgam/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set version = '1.0-3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-vgam
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: VGAM_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/VGAM_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/VGAM/VGAM_{{ version }}.tar.gz
+
+
+  sha256: 23bb6690ae15e9ede3198ef55d5d3236c279aa8fa6bd4f7350242379d9d72673
+  # patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: VGAMdata, MASS, mgcv
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
```

Comment:
At least for r packages yes. Also make sure to use conda-build 2.1.9, it supports the conda-forge style of r recipes, including a sha256 checksum.